openSUSE-2017.7.2/salt.spec

```diff
@@ -591,12 +591,18 @@ mv _build.python3 build
 %{__python3} setup.py --salt-transport=both install --prefix=%{_prefix} --root=%{buildroot}
 mv build _build.python3
 %endif
-%if 0%{?build_py2} && !0%{?default_py3}
-mv _build.python2 build
-rm -f %{buildroot}%{_bindir}/*
-%{__python} setup.py --salt-transport=both install --prefix=%{_prefix} --root=%{buildroot}
-mv build _build.python2
+
+%if 0%{?default_py3}
+DEF_PYPATH=_build.python3/scripts-*/
+%else
+DEF_PYPATH=_build.python2/scripts-*/
```

Comment:
Wondering if we can reuse this:
```specfile
%if 0%{?suse_version} >= 1320
%global py_major 3
%else
%global py_major 2
%endif
```

So then we could so something like:
```specfile
Requires:       python%{py_major}-CherryPy >= 3.2.2
...
DEF_PYPATH=_build.python%{py_major}/scripts-*/
```

Reply:
Will not work, as there is no python2-CherryPy. There is only a python-CherryPy .